Conversions/BinaryToDecimal.js

```diff
@@ -1,12 +1,11 @@
 function binaryToDeicmal (binaryNumber) {
-    let decimalNumber = 0
-    const binaryDigits = binaryNumber.split('').reverse()
-    binaryDigits.forEach((binaryDigit, index) => {
-      decimalNumber += binaryDigit * (Math.pow(2, index))
-    })
-    console.log(`Decimal of ${binaryNumber} is ${decimalNumber}`)
-  }
-  
-  binaryToDeicmal('111001')
-  binaryToDeicmal('101')
-  
\ No newline at end of file
+  let decimalNumber = 0
+  const binaryDigits = binaryNumber.split('').reverse()
+  binaryDigits.forEach((binaryDigit, index) => {
```

Comment:
@neha-saggam please add some hints and comment for better understanding
thank you,
@itsvinayak

Reply:
@itsvinayak I personally feel the code should be enough self-explanatory. Comments are considered as anti-patterns.
I have still added some comments but not really convinced of why do we need it. Please let me know if I am missing something here. :)